source/_integrations/spotify.markdown

```diff
@@ -83,7 +83,7 @@ Restart your Home Assistant instance before continuing with the next step.
 This integration supports multiple Spotify accounts at once. You don't need to
 create another Spotify application in the Spotify Developer Portal and no
 modification to the `configuration.yaml` file is needed. Multiple Spotify
-accounts can be linked to a _single_ Spotify application.
+accounts can be linked to a _single_ Spotify application. You will have to add those accounts into the **Users and Access** section of your application in the Spotify Developer Portal.
```

Comment:
That is not needed?

Reply:
Oh, that is only for new apps :(

Reply:
Correct, else it will give out error 403.